src/magicbox/SuggestionsManager.ts

```diff
@@ -46,6 +60,10 @@ export class SuggestionsManager {
     return this.resultPreviewsManager.hasPreviews;
   }
 
+  private get focusedSuggestion() {
+    return find(this.currentSuggestions || <Suggestion[]>[], suggestion => suggestion.dom.classList.contains(this.options.selectedClass));
```

Comment:
`|| <Suggestion[]` should not be needed, underscore `.find` on null/undefined will do the exact same behaviour you have here.

Reply:
TIL, I'll remove that ternary default.